README.rst

```diff
@@ -1,86 +1,78 @@
+  * Multi-rate Hi-Fi functionality:
+    * Synchronous Sample Rate Converter function (SSRC)
+    * Asynchronous Sample Rate Converter function (ASRC)
+    * Conversion between 44.1, 48, 88.2, 96, 176.4 and 192 KHz input and output sample rates.
+    * 32 bit PCM input and output data in Q1.31 signed format.
+    * Optional output dithering to 24 bit using Triangular Probability Density Function (TPDF).
+    * Optimized for xCORE-200 instruction set with dual-issue and for the Vector Processing Unit for xcore.ai.
+    * Block based processing - Minimum 4 samples input per call, must be power of 2.
+    * Up to 10000 ppm sample rate ratio deviation from nominal rate (ASRC only).
+    * Very high quality - SNR greater than 135 dB (ASRC) or 140 dB (SSRC), with THD of less than 0.0001% (reference 1KHz).
+    * Configurable number of audio channels per SRC instance.
+    * Reentrant library permitting multiple instances with differing configurations and channel count.
+    * No external components (PLL or memory) required.
+  * Fixed factor functionality:
+    * Synchronous fixed factor of 3 downsample and oversample functions supporting either HiFi quality
+      or reduced resource requirements for voice applications.
+    * Synchronous fixed factor of 3 and 3/2 downsample and oversample functions for voice applications
+      optimized for the XS3 Vector Processing Unit.
+  * Asynchronous FIFO:
+    * Non-blocking thread-safe FIFO for use with ASRC systems.
+    * Built in phase detector and control loop to determine the ASRC conversion ratio
+    * Enables practical audio systems to be constructed using multiple clock domains and sample rates
+
+Known Issues
+************
+
+  * Synchronous fixed factor of 3 and 3/2 downsample and oversample functions for voice applications optimized for the XS3 Vector Processing Unit
+    currently overflow rather than saturate in cases where a full scale input causes a perturbation above full scale at the output.
+    To avoid this scenario, please ensure that the input amplitude is always 3.5 dB below full scale.
+    The overflow behavior of these SRC components will be replaced by saturating behavior (to match all other SRC components) in a future release.
 
- * Synchronous factor of 3 downsample function for use with voice optimized for XS3 (ff3_96t_ds)
- * Synchronous factor of 3 oversample function for use with voice optimized for XS3 (ff3_96t_us)
+Required Tools
+**************
 
- * Synchronous factor of 3/2 downsample function for use with voice optimized for XS3 (rat_2_3_96t_ds)
- * Synchronous factor of 3/2 oversample function for use with voice optimized for XS3 (rat_3_2_96t_us)
+  * XMOS XTC Tools: 15.3.0
 
-There are three different component options that support fixed factor of 3 up/downsampling. To help choose which one to use follow these steps:
+Required Libraries (dependencies)
+*********************************
 
- #. If HiFi quality (130 dB SNR) up/downsampling is required, use ds3 or os3.
```

Comment:
Has this gone into the main docs? AD was very insistent that a simple guide as to "which SRC to use" was included

Reply:
No, it probably should do though. Readme isn't the place for in-depth documentation!